desktop-widgets/tagwidget.cpp

```diff
@@ -89,14 +89,6 @@ void TagWidget::reparse()
 	if (pos.first >= 0 && pos.second > 0)
 		currentText = text().mid(pos.first, pos.second - pos.first).trimmed();
 
-	/*
-	 * Do not show the completer when not in edit mode - basically
-	 * this returns when we are accepting or discarding the changes.
-	 */
-	if (MainWindow::instance()->mainTab->isEditing() == false) {
```

Comment:
so, if edit mode no longer exists, why do we have an isEditing() function - which is checked in almost a dozen places in current master...?

Reply:
Edit mode still exists when editing cylinders and weights. Therefore the accesses in MainWindow are probably valid.

But in this particular case it was used to close the completer when exiting edit mode(?). Which effectively meant never showing it, because edit mode is not activated for the dive-notes tab.

Perhaps this needs to be reinstated in some other form (on focus out?), but this check is clearly bogus. At least it survived light testing.

Reply:
I noticed on my Mac that I didn't get a tag completer at all. I'll merge this because yes, this makes sense - and will investigate a bit more why I'm not seeing a completer.

Reply:
Two points:

1) The commit message is not entirely correct - I will fix it later.
2) The editMode will perhaps stay for some time, because it is used when editing the profile of a manually added dive. Making every planner action undoable ... ouch. Though we could move it away from the maintab to the profile and thus contain it better.

Reply:
I guess I was a bit pre-mature in merging it, then. Oh well. Too late to fix the commit message.

Reply:
Well... It's not totally wrong, just a bit imprecise. Should be OK.